lightly/active_learning/agents/agent.py

```diff
@@ -0,0 +1,79 @@
+from typing import *
+
+from lightly.active_learning.config.sampler_config import SamplerConfig
+from lightly.active_learning.scorers.scorer import Scorer
+from lightly.api.api_workflow_client import ApiWorkflowClient
+from lightly.api.bitmask import BitMask
+
+
+class ActiveLearningAgent:
+    """A basic class providing an active learning policy
+
+    Attributes:
+        api_workflow_client:
+            The client to connect to the api.
+        preselected_tag_id:
+            The id of the tag containing the already labeled samples, default: None == no labeled samples yet.
+        query_tag_id:
+            The id of the tag defining where to sample from, default: None resolves to initial_tag
+
+    """
+
+    def __init__(self, api_workflow_client: ApiWorkflowClient, query_tag_id: str = None, preselected_tag_id: str = None):
```

Comment:
Are we using the `tag_id` until tag-names are unique? Or what's the reasoning here?

Reply:
I solved that, now the tag_names are used